src/meson.build

```diff
@@ -183,6 +180,7 @@ if get_option('man')
   )
 endif
 
+if get_option('polkit')
```

Comment:
Pretty sure you'll need the same wrapper around fwupdmgr's executable too.

Reply:
I think it's going to be inviting bugs, particularly further down the road when fwupd grows new APIs that require upgrading both.

Reply:
True. I'll ifdef it.